### PR TITLE
#00004. Non-ascii to ascii decoder

### PR DESCRIPTION
**Pull request by DeaThunder**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,6 +27,8 @@ Copies the local player's position and rotation onto the GameObject with the Udo
 
 Very messy script, check source code to see if it could be useful to you. Main features being `DateTime.Now.ToString()` formatting and `formatCustom` will replace `%playerLocal%` with the local player's display name.
 
+**For being able to use `forceAscii` feature, requires assigning ``Unidecoder`` object through a reference!**
```
Comment (Zyoh) [inline README.md]: Something like this seems easier to understand.
> ** `forceAscii` requires a game object containing `Unidecoder.cs` assigned to the `unidecoder` variable.

File: README.md [hunk 2]
```diff
@@ -35,4 +37,16 @@ Players within range of the game object will have their jump power increased. Po
 
 #### > OnInteractAnimationPlayer.cs
 
-Allows players to trigger animations from assigned `animator` via interacting with object, which has current script in it.
\ No newline at end of file
+Allows players to trigger animations from assigned `animator` via interacting with object, which has current script in it.
```
Comment (Zyoh) [inline README.md]: Could be written like this.
> Triggers the assigned animator when interacting with the object containing the script.

File: TextFormatting.cs [hunk 3]
```diff
@@ -21,14 +24,20 @@ public class TextFormatting : UdonSharpBehaviour
     [Tooltip("C# DateTime.Now.ToString formatting.")]
     public bool formatDateTime = false;
     [Tooltip("Replace Unicode characters with similar ASCII if available.")]
+    [Tooltip("Replace Unicode characters with similar ASCII if available. Requires Unidecoder for work.")]
```
Comment (Zyoh) [inline TextFormatting.cs]: "Requires Unidecoder for work." should be "Requires Unidecoder to work."

**Review by Zyoh — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (DeaThunder) [reply README.md]: Changed

File: TextFormatting.cs (hunk 3 above)
Comment (DeaThunder) [reply TextFormatting.cs]: Done

File: README.md (hunk 2 above)
Comment (DeaThunder) [reply README.md]: Fixed

**Review by Zyoh — APPROVED** [review]

**Pull request merged**